src/Simplex/Chat/Store.hs

```diff
@@ -4107,88 +4094,62 @@ setChatItemTTL db User {userId} chatItemTTL = do
         "INSERT INTO settings (user_id, chat_item_ttl, created_at, updated_at) VALUES (?,?,?,?)"
         (userId, chatItemTTL, currentTs, currentTs)
 
-getExpiredFileInfo :: DB.Connection -> User -> UTCTime -> IO [CIFileInfo]
-getExpiredFileInfo db User {userId} expirationDate =
+getChatsList :: DB.Connection -> User -> IO [ChatRef]
```

Comment:
this function can be unnecessary - why can't we just load contacts and groups separately (there are functions for that already I think? although they might be loading some extra data) and then to process in two different loops. Here we combine them into one type first and then pattern match back.